src/main/java/uk/gov/hmcts/reform/professionalapi/controller/internal/ProfessionalUserInternalController.java

```diff
@@ -54,22 +53,29 @@
     @ApiParam(
             name = "showDeleted",
             type = "string",
-            value = "flag (True/False) to decide deleted users needs to be shown",
-            required = false
+            value = "Flag (True/False) to decide whether Deleted Users are included in the response"
     )
     @ApiResponses({
             @ApiResponse(
                     code = 200,
-                    message = "List of a professional users along with details",
-                    response = ProfessionalUsersEntityResponse.class
+                    message = "List of Professional Users and their details",
+                    response = ProfessionalUsersResponse.class

```

Comment:
shouldn't it be ProfessionalUsersEntityResponse ?

Reply:
ProfessionalUsersEntityResponse does not provide an Example Model in Swagger, it only shows as:

{}

which is not useful, so using ProfessionalUsersResponse to provide a more useful Example Model

Reply:
use responseContainer = "list" and class as ProfessionalUsersResponse

Reply:
Thanks that worked and has been implemented in this commit: https://github.com/hmcts/rd-professional-api/pull/503/commits/112cdd4947432ed0c289a23196b9343ab71be578